packages/portalnetwork/src/wire/utp/Protocol/read/UtpReadListener.ts

```diff
@@ -0,0 +1,34 @@
+// import { Worker,  } from 'worker_threads';
+import cluster, {Worker} from 'cluster'
```

Comment:
Were you able to get this to compile and run in browser?  This is a nodejs module and looks like it's not available in the browser, at least according to [this PR](https://github.com/facebook/create-react-app/pull/6010)